nixos/modules/services/desktops/pipewire/wireplumber.nix

```diff
@@ -8,15 +8,18 @@ in
 
   options = {
     services.pipewire.wireplumber = {
-      enable = lib.mkEnableOption "A modular session / policy manager for PipeWire";
+      enable = lib.mkOption {
+        type = lib.types.bool;
+        default = config.services.pipewire.enable;
+        defaultText = lib.literalExpression "config.services.pipewire.enable";
+        description = "A modular session / policy manager for PipeWire";
```

Comment:
```suggestion
        description = "Whether to enable a modular session / policy manager for PipeWire";
```

Reply:
note that `mkEnableOption` already add the above wording unlike `mkOption`

Reply:
Applied, thanks.